goratings/math/glicko2.py

```diff
@@ -53,10 +53,9 @@ def expand_deviation_because_no_games_played(self, n_periods: int = 1) -> "Glick
         global MAX_RD
         global MIN_RD
 
-        for _i in range(n_periods):
-            phi_prime = sqrt(self.phi ** 2 + self.volatility ** 2)
-            self.deviation = min(MAX_RD, max(MIN_RD, GLICKO2_SCALE * phi_prime))
-            self.phi = self.deviation / GLICKO2_SCALE
+        phi_prime = sqrt(self.phi ** 2 + n_periods * self.volatility ** 2)
```

Comment:
Does python operator precedence get this right, or do we need parentheses around `self.volatility ** 2`?

Reply:
It gets it right :)